nubi/src/components/recargas/recargas.js

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+
+import RegisterButton from '../buttons/register/button';
+import logos from '../../assets/images/logos';
+
+import styles from './styles.module.scss';
+
+function Recargas() {
+  return (
+    <div className={styles.container}>
+      <div className={styles.subContainer}>
+        <h1 className={styles.title}> Recargá tu cuenta PayPal</h1>
+        <h2 className={styles.text}>
+          Con Nubi agregá dinero a tu cuneta PayPal desde tu banco y comprá en todo el mundo sin tarjeta de
+          crédito. ¡Si no tenés cuenta PayPal creátela desde Nubi en unos simple pasos!
+        </h2>
+        <h2 className={styles.text}>El costo de la recarga es del 5% + IVA sobre la tarifa.</h2>
+        <RegisterButton />
+      </div>
+      <h1 className={styles.subTitle}>Ahorrá en tus compras online</h1>
+      <div className={styles.imagesContainer}>
+        {logos.map(({ src, title }) => (
+          // eslint-disable-next-line react/jsx-key
+          <div className={styles.imageContainer}>
```

Comment:
Procura no omitir las keys cuando iteres sobre un array. Éstas ayudan a identificar el elemento en los algoritmos de diferenciación que usa React para saber qué reemplazar y qué no cuando hay un cambio en el código. Puedes leer más acerca de las keys [acá](https://reactjs.org/docs/lists-and-keys.html).

Reply:
Ahi solucione el problema con las keys (Gracias por la documentacion!)

Reply:
¡Esooo!